adapter/internal/operator/controllers/dp/tokenissuer_controller.go

```diff
@@ -102,11 +106,84 @@ func NewTokenIssuerReconciler(mgr manager.Manager) error {
 		loggers.LoggerAPKOperator.ErrorC(logging.PrintError(logging.Error2656, logging.BLOCKER, "Error watching TokenIssuer resources: %v", err.Error()))
 		return err
 	}
+	conf := config.ReadConfigs()
+	predicates := []predicate.Predicate{predicate.NewPredicateFuncs(utils.FilterByNamespaces(conf.Adapter.Operator.Namespaces))}
```

Comment:
don't we have to add this predicate to &dpv1alpha1.TokenIssuer{} in https://github.com/wso2/apk/pull/2032/files#diff-8dede92e4f838b72c1c5c1849a86b3b9efe6d4b67a5af3e138e4954f99bbf260R104 as well?